wolfhsm/wh_server.h

```diff
@@ -129,6 +129,13 @@ typedef enum {
     WH_DMA_OPER_CLIENT_WRITE_POST = 3,
 } whServerDmaOper;
 
+#ifdef WOLFHSM_CFG_DMA_CUSTOM_CLIENT_COPY
+typedef enum {
+    WH_DMA_OPER_CLIENT_READ = 0,
+    WH_DMA_OPER_CLIENT_WRITE = 1,
```

Comment:
Clarify in the name that these are only useable for the copy callback (don't want users passing them to regular callbacks). Sure wish we had real typed enums to work with...
```suggestion
    WH_DMA_COPY_OPER_CLIENT_READ = 0,
    WH_DMA_COPY_OPER_CLIENT_WRITE = 1,
```